fastlane/Fastfile.swift

```diff
@@ -55,4 +55,22 @@ class Fastfile: LaneFile {
         // TODO: - Make release notes
         Distribution.uploadToFirebase(environment: .production, releaseNotes: "")
     }
+    
+    // MARK: - Private Helper
+
+    private func setAppVersion() {
+        desc("check if any specific version number in build environment")
+        if !Constant.manualVersion.isEmpty {
+            incrementVersionNumber(
+                versionNumber: .userDefined(Constant.manualVersion)
+            )
+        }
+    }
+
+    private func bumpBuild() {
+        desc("set build number with number of commits")
+        incrementBuildNumber(
+            buildNumber: .userDefined(String(numberOfCommits())),
+            xcodeproj: .userDefined(Constant.projectPath))
+    }
```

Comment:
Don't seem to use these, could you explain more for they are for?

Reply:
@nkhanh44 What do you think if we have `buildNumber` as parameter?

```swift
    private func bumpBuild(buildNumber: Int = numberOfCommits() ) {
            ...
                buildNumber: .userDefined(String(buildNumber)),
            ...
    }
```

And could we use `bumpBuild()` at our `build` lanes?

Reply:
@blyscuit I created `setAppVersion` and `bumpBuild` functions match [ruby implement](https://github.com/nimblehq/ios-templates/blob/5664de6fc4a9da259295c4c1a1d3841d69b1dd72/fastlane/Fastfile#L210). They're used for set specific version number in build environment and bump it before build app and upload.
 I added them to build lanes such as `buildStagingAndUploadToFirebaseLane()`, `buildProductionAndUploadToFirebaseLane()`, `buildAndUploadToAppStoreLane()` 🙏 .

@blyscuit @vnntsu  I updated at https://github.com/nimblehq/ios-templates/pull/362/commits/68a1a4b202f5d4618c00f973183cb2c433848d1a and https://github.com/nimblehq/ios-templates/pull/362/commits/baad582d7c4b2ce20b40a68f956f0d6956d07546. Thank youu 🙏 .